### DISCUSSION TITLE
Multiplayer sessions between different users on SSH

### DISCUSSION THREAD

**Discussion by MasterAwesome** [zellij-org/zellij#3468] [Q&A] [answered]
I was reading [https://zellij.dev/news/multiplayer-sessions/](https://zellij.dev/news/multiplayer-sessions/) and it mentions a way for users to share terminal sessions but this seems to work only if it's the same user. Is there a way to share zellij sessions across users on the same machine?

# Things I tried
It looks like the sessions are stored as a socket at `XDG_RUNTIME_DIR/zellij/<version>/<session_name>`. Even if I create an folder that's shared between users and set my `XDG_RUNTIME_DIR` for zellij it chmods it to `700` at init and the non-owner of the socket cannot use it and crashes due to [this unwrap](https://github.com/zellij-org/zellij/blob/8e33b20559ecddcaba17e5cc8d6f05b638718cac/zellij-client/src/lib.rs#L227).

**Comment by giyany** [comment, marked as answer]
The short answer is: not yet for security reasons.
Discussed in #3406 .
